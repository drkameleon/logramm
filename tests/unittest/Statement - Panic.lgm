out "Before panic";
panic "I'm panicking!";
out "After panic";