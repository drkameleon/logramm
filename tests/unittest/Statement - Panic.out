Before panic
Runtime Error | File: tests/unittest/Statement - Panic.lgm
              | Line: 2

              | User exception: I'm panicking!